### ISSUE TITLE
Accessibility: Color Contrast:  Text appearing on Command Pallete doesn't have standard color contrast ratio.

### ISSUE THREAD

**Issue by Sachin077** [Microsoft/monaco-editor#656]
monaco-editor version: 0.X.Y
Browser: Chrome/Edge/Firefox
OS: Windows 10
Steps or JS usage snippet reproducing the issue:
![capture](https://user-images.githubusercontent.com/7456601/33536736-f4bf5b7e-d8dc-11e7-9dd5-8529fd4efd12.PNG)
